activerecord/lib/active_record/migration/compatibility.rb

```diff
@@ -101,7 +101,7 @@ def create_table(table_name, options = {})
           if connection.adapter_name == "Mysql2"
             super(table_name, options: "ENGINE=InnoDB", **options)
           else
-            super
+            super(table_name, **options)
```

Comment:
Can you pick 69712e82c83f25f8923dd74f4c4cd81b75616630 from master?

Reply:
Done